src/core/input.c

```diff
@@ -11,7 +11,7 @@
 
 #include <inttypes.h>
 
-#define SECTION_NAME_MAX 128
+#define SECTION_NAME_MAX 50
```

Comment:
Does inih export a preprocessor define for this? If so, it might be a good idea to use that. Might pose an issue with includes though.